scripts/remove_boilerplate.py

```diff
@@ -122,24 +111,24 @@ def process_record(self, index_id: int, index, warc: WARCRecord):
             return
 
         # Escape paragraph for parsable XML
-        extracted_text = '\n\n'.join(
-            f'<p>\n{xml.sax.saxutils.escape(paragraph)}\n</p>'
-            for paragraph in paragraphs
-        )
-        if len(extracted_text) == 0:
+        escaped_paragraphs = [xml.sax.saxutils.escape(paragraph) for
+                              paragraph in paragraphs]
+        if len(escaped_paragraphs) == 0:
             logging.info(f'Nothing\'s left of {index.url} '
                          'after boilerplate removal')
             return
+        current_document = Document(
```

Comment:
Just `document`. We only have one, so...

Reply:
Ok.